openjdk.test.jck/src/test.jck/net/adoptopenjdk/stf/Jck.java

```diff
@@ -0,0 +1,968 @@
+		fileContent += "writeReport " + reportDir + ";\n";
+		
+		// Replacing with Double backslash in the .JTB file. As the harness requires this to run tests.
+		fileContent = fileContent.replace("\\\\", "\\"); 		// This replaces \\ with \.
+		fileContent = fileContent.replace("\\", "\\\\\\\\");	// This replaces \ with \\.  But ("\\", "\\\\") works. 
+		 
+		test.doWriteFile("Writing into jtb file.", newJtbFileRef, fileContent);
+	}
+
+	public void execute(StfCoreExtension test) throws Exception {
+		String comment = "Running JCK in " + testExecutionType + " way with Agent " + withAgent;
+		FileRef jtbFile = test.env().getResultsDir().childDirectory("JTB").childFile("stf_generated.jtb");
+		if (testExecutionType.equals("multijvm") && withAgent.equals("off")) {
+			StfProcess javatestAgent = null;
+			StfProcess rmiRegistry = null;
+			StfProcess rmid = null;
+			StfProcess tnameserv = null;
+			
+			String executeJar;
+			if ((jckVersion.contains("jck9") || jckVersion.contains("jck8"))  && (testSuite == TestSuite.RUNTIME) ) {
+					executeJar = "/lib/jtlite.jar" ;
+			} else {
+					executeJar = "/lib/javatest.jar" ;
+			}
+			
+			if ( (testSuite == TestSuite.RUNTIME) && (tests.contains("api/java_util") || tests.contains("api/java_net") || tests.contains("api/java_rmi")  || tests.contains("api/javax_management") 
+					|| tests.contains("api/org_omg") || tests.contains("api/javax_xml") || tests.contains("vm/jdwp")) ) {
+				javatestAgent = test.doRunBackgroundProcess("Starting javatest agent", "AGNT", ECHO_ON, ExpectedOutcome.neverCompletes(), test.createJavaProcessDefinition()
+						.addJvmOption("-Djavatest.security.allowPropertiesAccess=true -Djava.security.policy=" + test.env().findPrereqFile("jck/" + jckVersion + "/" + testSuiteFolder + "/lib/jck.policy").toString())
+						.addJarToClasspath(test.env().findPrereqFile("jck/" + jckVersion + "/" + testSuiteFolder + executeJar))
+						.addDirectoryToClasspath(test.env().findPrereqDirectory("jck/" + jckVersion + "/" + testSuiteFolder + "/classes"))
+						.runClass("com.sun.javatest.agent.AgentMain")
+						.addArg(" -passive")
+						);
+				
+				rmiRegistry = test.doRunBackgroundProcess("Starting rmiregistry", "RMI", ECHO_ON, ExpectedOutcome.neverCompletes(), test.createJDKToolProcessDefinition()
+						.setJDKToolOrUtility("rmiregistry")	);
+				
+				rmid = test.doRunBackgroundProcess("Starting rmid", "RMID", ECHO_ON, ExpectedOutcome.neverCompletes(), test.createJDKToolProcessDefinition()
+						.setJDKToolOrUtility("rmid")	
+						.addArg("-J-Dsun.rmi.activation.execPolicy=none " + "-J-Djava.security.policy=" + test.env().findPrereqFile("jck/" + jckVersion + "/" + testSuiteFolder + "/lib/jck.policy").toString())
+						);
+				
+				tnameserv = test.doRunBackgroundProcess("Starting tnameserver", "TNAM", ECHO_ON, ExpectedOutcome.neverCompletes(), test.createJDKToolProcessDefinition()
+						.setJDKToolOrUtility("tnameserv")	
+						.addArg("-ORBInitialPort 9876")
+						);
+			}
+			
```

Comment:
Do we need this commented out section of the code (line 321-331)? If not, please remove it.

Reply:
Removed commented out code